server/game/cards/02_SHD/events/CrippleAuthority.ts

```diff
@@ -0,0 +1,35 @@
+import AbilityHelper from '../../../AbilityHelper';
+import { EventCard } from '../../../core/card/EventCard';
+
+export default class CrippleAuthority extends EventCard {
+    protected override getImplementationId() {
+        return {
+            id: '3228620062',
+            internalName: 'cripple-authority',
+        };
+    }
+
+    public override setupCardAbilities() {
+        this.setEventAbility({
+            title: 'Draw a card. Each opponent who controls more resources than you discards a card from their hand',
+            immediateEffect: AbilityHelper.immediateEffects.simultaneous(
+                (context) => [
+                    AbilityHelper.immediateEffects.draw({
+                        target: context.source.controller,
+                        amount: 1
+                    }),
+                    ...context.game.getPlayers()
```

Comment:
Since there's only one opponent to consider could we just do this as a conditional? I know Twin Suns is theoretically a thing we could do at some point but even if so it's so far off that I'd rather just keep implementations as clean and simple as possible on the assumption of one opponent for now, and if we do get there I'll want to add a lot of helpers to avoid needing to write up complex logic like this directly in the card impls. As much as possible the effects should be one-liners, and any complexity hidden in the systems (except for the cards that have really wacky one-off mechanics).

Reply:
Sure, I changed it! The conditional is a bit verbose though, it takes the same amount of code to target a single player.